location.txt

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
```

Comment:
@ldragoon  We don't need this file in master

Reply:
Removed this file.